lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -28,14 +28,14 @@ def initialize(build_mechanism,
     end
 
     def deploy_cli_hook(parser)
```

Comment:
I still do not understand why we need a deploy hook for this, we AFAIK do no look for ci status during deploy. Correct me if I am wrong.

Reply:
deploy hock removed